configure.ac

```diff
@@ -1160,7 +1160,7 @@ AC_ARG_WITH([sce-script-dir],
 
 if test "x${sce}" = xyes; then
   AC_DEFINE([ENABLE_SCE], [1], [compilation of script check engine enabled])
-  CFLAGS="$CFLAGS -DSCE_SCRIPT_DIR=\"${with_sce_script_dir}\""
+  CFLAGS="$CFLAGS -DSCE_SCRIPT_DIR=\\\"${with_sce_script_dir}\\\""
```

Comment:
Please add a comment why the double escape is needed, it is not obvious.

Reply:
It's not obvious to me either. The project compiled on my ubuntu container with single escaped quotes, but failed to compile on CI (https://jenkins.open-scap.org/job/openscap-pull-requests/612/console - ```error: 'tmp' undeclared```) until the quotes were double escaped.

Reply:
This is interesting. I am not 100% sure about what the cause may be. People are quite confused in this discussion too:
https://stackoverflow.com/questions/2410976/how-to-define-a-string-literal-in-gcc-command-line/15227471
The more robust, but also more arcane solution would be to write some macro magic to stringify the incoming macro, but I am fine if you just include a comment saying that the double escape is a tricky manner, very likely required in connection with jenkins CI.
Last but not least, please add the `configure.ac` code changes to `ac_probes/configure.ac.tpl` (I usually use `vimdiff` to find the right place where to add the lines.)